package.json

```diff
@@ -10,17 +10,55 @@
     "dist",
     "dist.browser"
   ],
+  "homepage": "https://github.com/ethereumjs/ethereumjs-wallet",
+  "dependencies": {
+    "@ethereumjs/util": "^8.0.0-beta.1",
+    "@scure/base": "^1.1.1",
+    "ethereum-cryptography": "^1.1.0",
+    "js-md5": "0.7.3",
+    "uuid": "8.3.2"
+  },
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "@typescript-eslint/eslint-plugin": "^4.27.0",
+    "@typescript-eslint/parser": "^4.27.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-typestrict": "1.0.3",
+    "eslint-plugin-implicit-dependencies": "^1.0.4",
```

Comment:
What is the demarcation line on keeping the caret or remove? Can't completely see the pattern.

Reply:
fixed